testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -117,6 +117,11 @@ pub struct Counters {
     pub naka_signer_pushed_blocks: RunLoopCounter,
     pub naka_miner_directives: RunLoopCounter,
 
+    #[cfg(test)]
+    pub naka_miner_current_rejections: RunLoopCounter,
+    #[cfg(test)]
```

Comment:
These shouldn't be `cfg(test)`, because `RunLoopCounter` is defined as an empty struct if `cfg(test)` is false.